examples/NeurIPS2019-Learn-to-Move-Challenge/README.md

```diff
@@ -1,8 +1,9 @@
- 
+# The Winning Solution for the NeurIPS 2019: Learn to Move Challenge
```

Comment:
Please introduce the aim of this folder at the beginning:
The **PARL** team gets the first place in [Neurips reinforcement learning competition](https://www.aicrowd.com/challenges/neurips-2019-learn-to-move-walk-around), again! This folder contains our final submitted model and the code relative to the training process.

Reply:
done